.prettierignore

```diff
@@ -0,0 +1 @@
+cypress_repro.md
```

Comment:
`prettier` is looking inside of `.github`? I would have thought it was just looking in `frontend` and `enterprise`

```
// package.json:190
    "prettier": "prettier --write '{enterprise/,}frontend/**/*.{js,jsx,css}'",
```

Reply:
Oopsy... that would be my `"editor.formatOnSave": true,`, I guess 🥴
Thanks for spotting this. I'll remove `.prettierignore`.